### ISSUE TITLE
Checking features instead of access modes for non visual content

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#564]
In the epub techniques, the variable setup for `non_textual_content_images` in 4.2.2 is checking the wrong metadata property for the visual content indicators:

> /package/metadata/meta[@property="schema:accessibilityFeature" and (text()="chartOnVisual" ...

It should be checking `schame:accessMode` not `schema.accessibilityFeature`. The value tests could also be shortened to avoid all the "or" connectors:

`/package/metadata/meta[@property="schema:accessMode" and text() = ("chartOnVisual", "chemOnVisual", "diagramOnVisual", "mathOnVisual", "musicOnVisual", "textOnVisual")]`

**Comment by mattgarrish** [comment]
Related to testing for one of a possible values, the `textual_alternative_images` variable could similarly be shortened to:

`/opf:package/opf:metadata/opf:meta[@property="schema:accessibilityFeature" and text() = ("longDescription", "alternativeText", "describedMath")]`

(Using text() for these, but see #562 for why it might be better to change them.)
